service/contacts.js

```diff
@@ -0,0 +1,63 @@
+const Contact = require("../models/contact.model");
+
+const listContacts = async () => {
+  const contacts = await Contact.find();
+  return contacts;
+};
+
+const getContactById = async (contactId) => {
+  const contact = await Contact.findOne({ _id: contactId });
+  if (contact) {
+    return contact;
+  } else {
+    throw new Error("Contact not found");
```

Comment:
Tu lepiej po prostu zwrócić  contact i w nadrzędnej funkcji jeśli jest nullem dać `return res.status(404.....` bo takie sprawdzanie po wiadomości to średni pomysł.